Sources/Classes/RSUtils.m

```diff
@@ -192,6 +192,18 @@ + (NSString* _Nullable) getBase64EncodedString:(NSString* __nonnull) inputString
     return base64EncodedString;
 }
 
++ (NSNumber * _Nullable) convertStringIntoNSNumber:(NSString* __nonnull) stringNumber {
+    NSNumberFormatter *numberFormatter = [[NSNumberFormatter alloc] init];
+    NSNumber *number = [numberFormatter numberFromString:stringNumber];
+    return number;
+}
+
++ (BOOL) isApplicationUpdated {
```

Comment:
Can you use this method for Life cycle manager class?

Reply:
I've utilised this method in the Application LifeCycle Manager class.